### ISSUE TITLE
Document how to use backticks in inline code

### ISSUE THREAD

**Issue by claremacrae** [obsidianmd/obsidian-help#492]
This section documents how to use backticks in a line:

https://help.obsidian.md/Editing+and+formatting/Basic+formatting+syntax#Inline+code

It would be helpful if it could also explain how to put a backtick inside inline code.

Since I just found out how, after checking the Obsidian docs, I thought I would share the answer...

###  Solution

From searching I found this:

https://meta.stackexchange.com/a/82722

And the double backticks works in Obsidian.

```text
- `hello`
- `hello \`world`
- ``hello`world``
```

displays as:

<img width="180" alt="image" src="https://github.com/obsidianmd/obsidian-help/assets/4840096/b375c31d-87c8-4bcc-bee0-86f3e7aab4f0">

In other words, using two backticks at either end of inline code snippet allows putting backticks inside, in Obsidian 1.3.5.

**Comment by claremacrae** [comment]
Also, if you have 2 backticks in the inline code, you can surround with 3:

```
- ```group by function '``' + task.originalMarkdown + '``'```
```

Gives:

![image](https://github.com/obsidianmd/obsidian-help/assets/4840096/3d44999f-1ecf-4825-9e31-201a4c688d18)

Very cool!!!

**Comment by marcusolsson** [comment]
Thank you for the feedback! I think that's definitely worthy of an entry in the docs! 🙏

**Comment by ericaxu** [comment]
Added in https://github.com/obsidianmd/obsidian-help/commit/384dd2bdae8d6cc6883880722ccfb2325db4735a
